python/nano/test/test_trainer_ipex.py

```diff
@@ -15,24 +15,25 @@
 #
 
 
-import pytest
 import os
 from unittest import TestCase
 
+from pytorch_lightning import LightningModule
+import pytest
 import torch
+import torchmetrics
+from test._train_torch_lightning import create_data_loader, data_transform
+from test._train_torch_lightning import train_with_linear_top_layer
```

Comment:
why do we need these import changes?

Reply:
It's used below to create a train_loader. And IDE sort the imports automatically.